src/apps/Console/Console.js

```diff
@@ -328,30 +306,21 @@ function Prompt({
         adornmentPosition="start"
         onChange={e => handleChange(e.target.value)}
         onKeyDown={e => {
-          if (e.keyCode === ENTER_KEY && !isDisabled()) {
-            setCommandHistory([...commandHistory, command])
+          if (e.keyCode === KEYCODES.enter && !isDisabled) {
+            const newCommandHistory = [...commandHistory, command]
+            setCommandHistory(newCommandHistory)
+            setHistoryIndex(newCommandHistory.length - 1)
```

Comment:
Actually... now I'm wondering; should we use `newCommandHistory.length` (because pressing `up` should show the last command, rather than the previous command)?

Reply:
I believe not, as historyIndex is contained in the range `0...commandHistory.length - 1`, for accesing the `commandHistory` array.